Makefile

```diff
@@ -146,16 +146,21 @@ generate-crds: vendor
 			docker; \
 	fi
 
+ifeq ($(DOCKER),podman)
+DOCKER_OPTS=-v $(PWD):$(PWD):Z
```

Comment:
Do you happen to have a doc link to describe why this change is required?

Reply:
I can't find it right now, but `--user` works differently with podman, and `:z` (`:Z`) is required to write to a host directory via a volume.